python/ray/_private/worker.py

```diff
@@ -2903,7 +2889,7 @@ def get(
             corresponding object becomes available. Setting ``timeout=0`` will
             return the object immediately if it's available, else raise
             GetTimeoutError in accordance with the above docstring.
-        _tensor_transport: [Alpha] The tensor transport to use to fetch `torch.Tensors` found in the Ray Direct Transport object. Currently, this supports "object_store" and "nixl".
+        _fetch_through_object_store: [Alpha] To fetch an RDT object through the object store instead of using its designated tensor transport. You may want to do this in situations where you want the value on driver but maybe the driver is not part of the tensor transport's collective.
```

Comment:
```suggestion
        _fetch_through_object_store: [Alpha] To fetch an RDT object through the object store instead of using its designated tensor transport. Set to True for cases where the caller does not support the object's tensor transport, e.g., the tensor transport is "nccl" and the caller is not part of the collective.
```